.jenkins

```diff
@@ -2,15 +2,15 @@
 
 
 // Include this shared CI repository to load script helpers and libraries.
-library identifier: 'vapor@1.19.7', retriever: modernSCM([
+library identifier: 'vapor@craig/VIO-1911', retriever: modernSCM([
```

Comment:
A question because I don't understand how the first parameter works here. This looks like a git tag, but based on the repo does not seem to be. What is it?

Reply:
This is a git tag for the ci-shared repository.
Specifically this: https://github.com/vapor-ware/ci-shared/tree/craig/VIO-1911

I am temporarily switching to this branch, so we can get some testing in, and if you could give me
feedback that things are working with this go build, I would appreciate it.

Subsequent to this, I will merge my changes in the ci-shared branch,
and come back to this repository and change this identifier to a proper tag on the ci-shared branch.

Thanks.